Applications/Spire/Source/Ui/OpenFilterPanel.cpp

```diff
@@ -294,9 +294,10 @@ bool OpenFilterPanel::event(QEvent* event) {
   if(event->type() == QEvent::ShowToParent) {
     m_filter_panel->show();
     window()->activateWindow();
-    m_input_box->setFocus();
   } else if(event->type() == QEvent::HideToParent) {
     m_filter_panel->hide();
+  } else if(event->type() == QEvent::FocusIn) {
```

Comment:
Why not use a focus proxy?

Reply:
After testing, the focus proxy could break the Tab order. When the input box is set to the focus proxy of the OpenFilterPanel, the two Radio buttons on the top can't get focus by pressing TAB key.